### PR TITLE
Raw HTTP Gsuite Connector Edited

### PR DESCRIPTION
**Pull request by sscode52**
Incorporated changes from the previous PR for the same connector #51

### CODE REVIEW

File: workflows/raw_http_connection_gsuite/workflow.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+   "name": "raw_http_connection_gsuite",
+   "title": "Raw HTTP Connector to GSuite",
+   "description": "Our current Workflows connector for Gsuite Admin does not include all functions within Google Workspace. The Custom API card is also restricted to the Directory and Licensing APIs. If there are other actions that you’d like to execute the Google has exposed an API for, we can use this guide to create a raw HTTP connector and use that to request the scopes needed.",
```
Comment (charleschan-okta) [inline workflows/raw_http_connection_gsuite/workflow.json:4]: Actually Google Workspace app is used in the flow, so pl add:
  "connectors": [
    "googledirectory"
  ],

**Review by charleschan-okta — CHANGES_REQUESTED** [review]
Please add "googledirectory" connectior

File: workflows/raw_http_connection_gsuite/workflow.json (hunk 1 above)
Comment (sscode52) [reply workflows/raw_http_connection_gsuite/workflow.json:4]: I've updated this

**Review by charleschan-okta — APPROVED** [review]
LGTM 🚀

**Pull request merged**